docs/resources/yaml.md.erb

```diff
@@ -11,36 +11,36 @@ Use the `yaml` InSpec audit resource to test configuration data in a Yaml file.
 
 ## Syntax
 
-A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file:
+A `yaml` resource block declares the configuration data to be tested. Assume the following Yaml file called 'filename.yml':
 
-    name: foo
-    array:
-      - zero
-      - one
+    name: myservice
+    hosts:
+      - ip1.host.com
+      - ip2.host.com
 
-This file can be queried using:
+This file can be queired using:
 
     describe yaml('filename.yml') do
-      its('name') { should eq 'foo' }
-      its(['array', 1]) { should eq 'one' }
+      its('name') { should eq 'myservice' }
+      its(['hosts', 0]) { should eq 'ip1.host.com' }
     end
 
 where
 
-* `name` is a configuration setting in a Yaml file
-* `should eq 'foo'` tests a value of `name` as read from a Yaml file versus the value declared in the test
```

Comment:
Again, let's stay with the simpler example. However, the second bullet point could be: 

-* `should eq 'foo'` tests the value of `name` as read from a Yaml file against the value declared in the test.